modules/knative-service-cluster-local.adoc

```diff
@@ -12,16 +12,21 @@ Publicly accessible URLs are accessible from outside of the cluster.
 However, developers may need to build back-end services that are only be accessible from inside the cluster, known as _private services_.
 // Cluster administrators can configure private services for the cluster so that all services are private by default.
 // Need to add additional details about editing the configmap for admins
-Developers can label individual services in the cluster with the `serving.knative.dev/visibility=cluster-local` label to make them private.
+Developers can label individual services in the cluster with the `networking.knative.dev/visibility=cluster-local` label to make them private.
 
 .Procedure
 
-* Set the visibility for your service by adding the `serving.knative.dev/visibility=cluster-local` label:
+[IMPORTANT]
+====
+Both `serving.knative.dev/visibility` and `networking.knative.dev/visibility` labels are available and same functionality but `serving.knative.dev/visibility` will not be supported in the future release.
```

Comment:
I wrote this but I wonder if this note is not necessary. 
The note needs to be removed when serverless v1.15 is released :sweat_smile:

Reply:
Yeah I wouldn't bother adding it for just one release I think, I'd just add the change to release notes and update the label in docs as you have done.
Let's not merge this one until 1.15.0 either if it's not valid until the next release